app/models/outgoing_message.rb

```diff
@@ -74,6 +74,25 @@ class OutgoingMessage < ActiveRecord::Base
     self.default_url_options[:protocol] = "https"
   end
 
+  def self.expected_send_errors
+    [ EOFError,
+      IOError,
+      Timeout::Error,
+      Errno::ECONNRESET,
+      Errno::ECONNABORTED,
+      Errno::EPIPE,
+      Errno::ETIMEDOUT,
+      Net::SMTPAuthenticationError,
+      Net::SMTPServerBusy,
+      Net::SMTPSyntaxError,
+      Net::SMTPUnknownError,
+      OpenSSL::SSL::SSLError ].concat(additional_send_errors)
+  end
+
+  def self.additional_send_errors
+    []
```

Comment:
I'm assuming the expectation here is sites that need to define extras would do so by overriding this in their theme?

Reply:
Done so that you can either (as you have) define the extras and keep the rest of the defaults or redefine the lot if you really need to